docs/user/advanced.rst

```diff
@@ -274,15 +274,13 @@ If you specify a wrong path or an invalid cert, you'll get a SSLError::
 CA Certificates
 ---------------
 
-By default, Requests bundles a set of root CAs that it trusts, sourced from the
-`Mozilla trust store`_. However, these are only updated once for each Requests
-version. This means that if you pin a Requests version your certificates can
-become extremely out of date.
-
-From Requests version 2.4.0 onwards, Requests will attempt to use certificates
-from `certifi`_ if it is present on the system. This allows for users to update
-their trusted certificates without having to change the code that runs on their
-system.
+Requests uses certificates from the package `certifi`_. This allows for users
+to update their trusted certificates without changing the version of Requests.
+
+Before version 2.16, Requests bundled a set of root CAs that it trusted,
+sourced from the `Mozilla trust store`_. The certificates were only updated
+once for each Requests version. This meant that the certificates of a pinned
+version of Requests could become extremely out of date.
```

Comment:
They would be out of date if and only if users didn't separately install certifi. This is a blanket statement that is plainly untrue.

Reply:
This is why I included "could". It is true under some conditions, but not others.

How would you like to see it altered?

What about?

> This meant that the certificates of a pinned version of Requests could become extremely out of date if certi was not installed.

Or should the sentence be removed entirely? Any other suggestions?

Reply:
I think the `could` is too subtle. What about:

```
When `certifi` was not installed, this led to extremely out-of-date certificate bundles when using significantly older versions of Requests.
```